src/Symfony/Bundle/FrameworkBundle/Resources/config/templating_php.xml

```diff
@@ -34,8 +34,7 @@
 
         <service id="templating.helper.assets" class="Symfony\Bundle\FrameworkBundle\Templating\Helper\AssetsHelper">
```

Comment:
Yep, and we have this check in the `FrameworkExtension` https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L543-L547